frontend/pages/article/[slug].tsx

```diff
@@ -38,7 +38,7 @@ const Article: NextPage = () => {
         data && <ModalShare
           isVisible={isCopyModalVisible}
           onCancel={() => setCopyModalVisible(false)}
-          textLink={data?.url}
+          textLink={data?.url.slice(0, 30) + "..."}
```

Comment:
I think that this is not exactly what Paula was commenting on, I think the idea is to keep only the base url as shown in her example.

_Instead of containing the whole URL, the link text should be truncated to [subdomain].[domain].[top level domain] 
E.g. from “https://www.nytimes.com/2023/03/08/opinion/noam-chomsky-chatgpt-ai.html” to “[nytimes.com](http://nytimes.com/)”_

Reply:
ok, let me implement that.